roles/ceilometer-common/templates/etc/serverspec/ceilometer_spec.rb

```diff
@@ -0,0 +1,43 @@
+require 'spec_helper'
+
+describe file('/etc/ceilometer/ceilometer.conf') do
+  it { should contain(/.{8,}/).after(/^admin_password[ ]+=[ ]+/) }
+end
+
+dirs = ['/etc/ceilometer/', '/var/log/ceilometer/']
+dirs.each do |dir|
+  describe file(dir) do
+    it { should be_directory }
+    it { should be_mode '755'}
+  end
+end
+
+files = ['api_paste.ini', 'event_pipeline.yaml', 'policy.json',
+  'ceilometer.conf', 'event_definitions.yaml', 'pipeline.yaml']
+files.each do |file|
+  describe file("/etc/ceilometer/#{file}") do
+    it { should be_mode '644'}
+  end
+end
+
+describe process('ceilometer') do
+  its(:user) { should eq 'ceilometer' }
+end
+
+#some sorcery to get the all files in a directory, then remove ., and .. entries
+files = Dir.entries('/var/log/ceilometer')
+hash = Hash[files.map.with_index.to_a]
+files.delete_at(hash['.'])
+hash = Hash[files.map.with_index.to_a]
+files.delete_at(hash['..'])
```

Comment:
You can use `files = Dir['/var/log/ceilometer/*']` and it will not include the `.` or `..`.